pkg/install/0-installstorage.go

```diff
@@ -262,10 +242,37 @@ func (i *Installer) installStorage(ctx context.Context, installConfig *installco
 		return err
 	}
 
-	// the graph is quite big so we store it in a storage account instead of in cosmosdb
-	err = i.saveGraph(ctx, g)
+	// Ensure graph exists
+	g, err := i.loadGraph(ctx)
 	if err != nil {
```

Comment:
let's do this only on whatever specific "not found" error you get back from loadGraph; return err on anything else

Reply:
Changed it so it'll only create the graph if it gets a 404, as per https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob